### ISSUE TITLE
Windows + R: microsoft-edge:https://google.com

### ISSUE THREAD

**Issue by oleberdal-amedia** [rcmaehl/MSEdgeRedirect#388]
This opens google.com in Microsoft Edge, however I was under the impression it was supposed to be redirected to another browser (my default is Google Chrome).

**Comment by rcmaehl** [comment]
That is correct. DO you know which mode you installed the app using? Service Mode or Active Mode? Service Mode will need to be launched from the Start Menu as "MSEdgeRedirect" but Active Mode works automatically.

**Comment by oleberdal-amedia** [comment]
I don't know the difference between Service Mode or Active Mode, but I installed by typing `winget install MSEdgeRedirect` in Terminal (PowerShell).

**Comment by oleberdal-amedia** [comment]
Seems I want Active Mode instead of Service Mode?

Anyway half the time my Windows Menu web search clicks opens Google search in Google Chrome (correctly), and the other half it opens Bing search in Edge🤔🤔

 I have the following settings active:
- Hide Service Mode from Tray
- Hide Service Mode With Windows
- Bing Discover: Google
- Bing Images: Google
- Bing Search: Google
- PDF Viewer: Custom (chrome.exe)
- MSN News: (ALPHA) Google
- MSN Weather: AccuWeather
- Disable Windows CoPilot UNCHECKED
- Redirect Bing Chat CHECKED
- Redirect Windows Store 'Apps' CHECKED

**Comment by oleberdal-amedia** [comment]
SOLVED: Although this issue might properly reflect some problem with Service Mode, I no longer have any issues after using the Installer and using Active Mode.
